Cargo.toml

```diff
@@ -31,6 +31,7 @@ tray = [ "tao/tray" ]
 devtools = [ ]
 transparent = [ ]
 fullscreen = [ ]
+pictureInPicture = [ ]
```

Comment:
I don't think it needs a flag. Just add it as builtin until someone need it as attribute to disable.